pywbem/_logging.py

```diff
@@ -0,0 +1,402 @@
+# Copyright 2017 InovaDevelopment Inc. All Rights Reserved.
+
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+"""
+This package supports logging using the standard Python :mod:`py:logging`
+module. The logging support for multiple :class:`~py:logging.Logger` objects
+(named loggers):
+
+Currently there are two defined in this code:
+
+* 'pywbem.ops' for user-issued calls to pywbem CIM OperationsAPI functions, at
+  the info level. Internal calls to API functions are not logged.
+
+* 'pywbem.http' for http level information, at the info level.
```

Comment:
What does "at the info level" mean? If it means that these loggers produce log entries at info level or higher levels, we should say that in more specific words. This comment applies to both loggers.